src/ctia/stores/atom/common.clj

```diff
@@ -7,59 +7,50 @@
   (fn [_new-entity_]
     (str prefix "-" (UUID/randomUUID))))
 
-(defmacro def-read-handler [name Model]
-  `(s/defn ~name :- (s/maybe ~Model)
-     [state# :- (s/atom {s/Str ~Model})
-      id# :- s/Str]
-     (get (deref state#) id#)))
+(defn read-handler [Model]
+  (s/fn :- (s/maybe Model)
```

Comment:
You're right, but I don't see how I could use a `defn` without going back to using macros. How would you do this?